proxy/madara/Dockerfile

```
@@ -0,0 +1,8 @@
+FROM docker.io/uacias/madara:v1.0.0 
+
+USER root
```

Hadolint

[warning] Last USER should not be root

(DL3002)